modulemd/tests/ModulemdTests/upgradehelper.py

```diff
@@ -1,84 +0,0 @@
-#!/usr/bin/python3
```

Comment:
Just curious why you decided to drop this?

Reply:
I didn't decide to; I must have hit ctrl-d in the wrong window at some point. Thanks for noticing.

Reply:
Oh, ignore the above. I thought from the diff that I'd just clobbered the hashbang.

Since we moved the UpgradeHelper into a private object, it is no longer exposed to the Python bindings. Thus there are no python tests we can run.

Reply:
Got it. Thanks.